provisioning/service/src/Config/EnrollmentGroup.cs

```diff
@@ -258,14 +258,21 @@ public Attestation Attestation
             }
             set
             {
-                if(!((value ?? throw new ArgumentNullException(nameof(value))) is X509Attestation))
+                if (value == null)
                 {
-                    throw new ArgumentException("Attestation for enrollmentGroup shall be X509");
+                    throw new ArgumentNullException(nameof(value));
+                }
+                else if (!(value is X509Attestation) && !(value is SymmetricKeyAttestation))
+                {
+                    throw new ArgumentException("Attestation for enrollmentGroup shall be X509 or symmetric key");
```

Comment:
nit: Attestation for enrollmentGroup must be either X509 or symmetric key?

Reply:
Sure, I'll update the error message